src/main/java/com/palantir/docker/compose/execution/RetryingDockerCompose.java

```diff
@@ -64,6 +64,15 @@ public void rm() throws IOException, InterruptedException {
     }
 
     @Override
+    public void exec(DockerComposeExecOption dockerComposeExecOption, String containerName,
```

Comment:
Not sure if we should retry docker execs. I haven't seen them go wrong so much in practice as say creating a container.